src/ui/server/controllers/quote/buyer-country/index.ts

```diff
@@ -71,9 +71,9 @@ export const get = async (req: Request, res: Response) => {
   let mappedCountries;
 
   if (countryValue) {
-    mappedCountries = mapCountries(countries, countryValue.isoCode);
+    mappedCountries = mapCisCountries(countries, countryValue.isoCode);
```

Comment:
can we try catch the api call on line 59?

Reply:
Great spot - this controller was created ages ago (first page!) I've added a try/catch and improved test coverage by adding a unit test for when the API call/promise is rejected